### ISSUE TITLE
model weight

### ISSUE THREAD

**Issue by linchunmian** [jwchoi384/Gaussian_YOLOv3#46]
Hi, could you please upload model weight to repository? Sorry, I cannot download it from the google.drive.
Thanks!

**Comment by jwchoi384** [comment]
@ChunmianLin 
Due to capacity issues in GitHub, I can't upload the file to my repo directly.
Why you can't download by google drive?
This is working fine.

**Comment by linchunmian** [comment]
Thanks! I don’t know where is the problem, and the page always collaspes.  

发送自 Windows 10 版邮件应用

发件人: Jiwoong Choi
发送时间: 2019年12月23日 14:00
收件人: jwchoi384/Gaussian_YOLOv3
抄送: ChunmianLin; Mention
主题: Re: [jwchoi384/Gaussian_YOLOv3] model weight (#46)

@ChunmianLin
Due to capacity issues in GitHub, I can't upload the file to my repo directly.
Why you can't download by google drive?
This is working fine.

**Comment by linchunmian** [comment]
Hi, Jiwoong. The reason why I want the pretrained model weight is that when I trained on BDD100K from scratch, the loss stuck in NaN. I tried to decrease learning rate, but it always existed when model trained several hundred epochs. I just follow your training settings, but what is the problems? Please help me!
Thanks!

发送自 Windows 10 版邮件应用

发件人: Jiwoong Choi
发送时间: 2019年12月23日 14:00
收件人: jwchoi384/Gaussian_YOLOv3
抄送: ChunmianLin; Mention
主题: Re: [jwchoi384/Gaussian_YOLOv3] model weight (#46)

@ChunmianLin
Due to capacity issues in GitHub, I can't upload the file to my repo directly.
Why you can't download by google drive?
This is working fine.

**Comment by wangzizhe** [comment]
@ChunmianLin hi, did you solve this problem, for me, there are all nan during training...
